nearcore/src/lib.rs

```diff
@@ -317,7 +317,8 @@ pub fn start_with_config_and_synchronization(
             "Rosetta RPC",
             start_rosetta_rpc(
                 rosetta_rpc_config,
-                Arc::new(config.genesis.clone()),
+                config.genesis.clone(),
```

Comment:
```suggestion
                config.genesis,
```

I think this'll work? 

Also, while we are at it, might remove `use near_rosetta_rpc::start_rosetta_rpc;`